src/supermarket/app/controllers/cookbook_versions_controller.rb

```diff
@@ -25,6 +25,7 @@ def show
     @collaborators = @cookbook.collaborators
     @supported_platforms = @version.supported_platforms
     @owner_collaborator = Collaborator.new resourceable: @cookbook, user: @owner
+    @metric_results = @version.metric_results
```

Comment:
Oof. Yea. `.includes()` would need to move to the query on the model, somewhere in the onion layers of `Cookbook`.

I'd like to rephrase this review comment as "maybe we can refine this later." 😄